### ISSUE TITLE
Version with brackets breaks APA bibliography

### ISSUE THREAD

**Issue by mdoering** [michel-kraemer/citeproc-java#108]
When I render the bibliography for the following CSL-JSON

```
{
  "id": "ID1",

  "type": "chapter",
  "title": "Catalogue of the Alucitoidea of the World",
  "DOI": "10.1093/database/baw125",
  "author": [
    {
      "family": "Hobern",
      "given": "Donald"
    },
    {
      "family": "Gielis",
      "given": "C."
    }
  ],
  "editor": [
    {
      "family": "Hobern",
      "given": "Donald"
    }
  ],
  "container-title": "Catalogue of Life Checklist",
  "issued": {
    "date-parts": [
      [
        2021,
        7,
        18
      ]
    ]
  },
  "version": "1.0.21.199 (18 Jul 2021)"
}
```
with the 7th APA style from Zotero using Frank Bennetts citeproc-js v1.4.61 I yield:

> Hobern, D., & Gielis, C. (2021). Catalogue of the Alucitoidea of the World. In D. Hobern (Ed.), Catalogue of Life Checklist (1.0.21.199 (18 Jul 2021)). https://doi.org/10.1093/database/baw125

When I do the same with the latest v3 snapshot from citeproc-java the closing brackets from the version are omitted and some superflous ampersand is added to the single editor:

> Hobern, D., & Gielis, C. (2021). Catalogue of the Alucitoidea of the World. In D. Hobern (Ed.), & , Catalogue of Life Checklist (1.0.21.199 (18 Jul 2021). https://doi.org/10.48580/e456fgvzb

**Comment by michel-kraemer** [comment]
I've fixed the issue with the closing parenthesis in https://github.com/michel-kraemer/citeproc-java/commit/503d1fbf5b1a825443727a6901e5b1aea4808f64. The superfluous ampersand was already fixed in 6ee967d8db9b11797b708373df85e4734632fcd0.

**Comment by michel-kraemer** [comment]
This issue is now fixed in citeproc-java v3.0.0-alpha.3
